error-handling.js

```diff
@@ -1,3 +1,17 @@
 exports.pathNotFound = (req, res, next) => {
     res.status(404).send({msg: "path not found"});
 };
+
+exports.customError = (err, req, res, next) => {
+    if(err.status && err.msg){
+        res.status(err.status).send({msg: err.msg});
+    }
+    else next(err);
+};
+
+exports.badRequest = (err, req, res, next) => {
+    if(err.code === '22P02'){
+        res.status(400).send({msg: 'bad request'});
+    }
+    else next(err);
```

Comment:
Where would the error be passed onto? You should have some error handling in place to handle the server errors and also act as a debugger for any errors that are not caught by the two previous error handling middleware functions.